gflownet/gflownet.py

```diff
@@ -1010,6 +1018,7 @@ def test(self, **plot_kwargs):
                 self.kl,
                 self.jsd,
                 self.corr_prob_traj_rewards,
+                self.var_logrewards_logp,
                 self.nll_tt,
```

Comment:
```suggestion
                corr_prob_traj_rewards,
                var_logrewards_logp,
                nll_tt,
```
The classic.

Reply:
Thanks! The good fix would be actually different, but the good good fix will change all this anyway.